src/smclient/smclient.cpp

```diff
@@ -601,7 +601,7 @@ void SMClient::UpdatePBHandlerState()
         }
 
 #ifndef CONFIG_ZTEST
-        if (err = mTLSChannel.Init(*mCertHandler, *mCertLoader, *channel); !err.IsNone()) {
+        if (err = mTLSChannel.Init("sm", *mCertHandler, *mCertLoader, *channel); !err.IsNone()) {
```

Comment:
consider adding a const to hold "sm" value

Reply:
used only once for log purpose